config/service_providers.yml

```diff
@@ -180,16 +189,16 @@ production:
     attribute_bundle:
       - email
 
-  'urn:gov:gsa:SAML:2.0.profiles:sp:sso:rails-qa':
-    acs_url: 'https://sp.qa.login.gov/auth/saml/callback'
-    assertion_consumer_logout_service_url: 'https://sp.qa.login.gov/auth/saml/logout'
-    sp_initiated_login_url: 'https://sp.qa.login.gov/login'
+  'urn:gov:gsa:SAML:2.0.profiles:sp:sso:rails-demo':
+    acs_url: 'https://sp.demo.login.gov/auth/saml/callback'
+    assertion_consumer_logout_service_url: 'https://sp.demo.login.gov/auth/saml/logout'
+    sp_initiated_login_url: 'https://sp.demo.login.gov/login'
     block_encryption: 'aes256-cbc'
     cert: 'sp_rails_demo'
     agency: 'A Gov Agency'
     friendly_name: 'Demo SP Application'
     logo: 'generic.svg'
-    return_to_sp_url: 'https://sp.qa.login.gov'
+    return_to_sp_url: 'https://sp.demo.login.gov'
```

Comment:
Do we need this? Didn't we get rid of the demo env a long time ago?

Reply:
Good point. I was just organizing what was already there but we should remove the demo env since it's no longer used. I'll add that to this PR.